src/components/navBar/navBar.module.scss

```diff
@@ -79,4 +78,106 @@ $theme-active: #87d870;
         color: $white;
         font-weight: 600;
     }
+
+    .hamburgerIcon {
+        display: none;
+    }
+    ul li {
+        list-style: none;
+    }
+    .navLinks ul {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 1rem;
+    }
+    .active {
+        color: $theme-active;
+        text-decoration: underline;
+        text-decoration-thickness: $thickness;
+        text-underline-offset: $offset;
+    }
+}
+@media (max-width: 970px) {
+    .navBar {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        position: relative;
+        .navLinks {
+            a {
+                padding: 10px 40px;
+                text-align: left;
+                color: $theme-blue;
+                display: none;
+            }
+        }
+    }
+
+    .signInLink {
+        position: absolute;
+        top: 10px;
+        right: 20px;
+    }
+
+    .showLogo {
+        display: none !important;
+    }
+
+    .hamburgerIcon {
+        display: inline-block !important;
+        height: 73px;
+        width: 73px;
+        padding: 18px;
+        cursor: pointer;
+        font-size: 20px;
+        color: $white;
+    }
+    .icon {
+        height: 32px;
+    }
+
+    .navBarMenu {
+        background: $white;
+        display: inline-block;
+        width: 100%;
+    }
+
+    .navBarMenu ul {
+        display: flex;
+        flex-direction: column;
+        box-shadow: 0px 10px 15px rgb(0 0 0 / 50%);
+        width: 100%;
+    }
+
+    .navBarMenu li {
+        padding: 12px 40px;
+        font-size: 16px;
+        text-align: left;
+        font: 700 16px Roboto, sans-serif;
+    }
+
+    .navBarMenu li a {
```

Comment:
use classes instead of this ul, li, a

Reply:
Need to use element selectors, because li and a tag are repetitive in the code and If I use className for li and a tag then it will make the HTML file look boring and dull, I think using element selectors can be useful sometimes in these conditions.